containerd-shim-wws-v1/src/main.rs

```diff
@@ -138,7 +138,7 @@ impl Instance for Workers {
                     let routes = Routes::new(&path, "", Vec::new(), &config);
 
                     // Final server
-                    let f = serve(&path, routes, WWS_ADDR, WWS_PORT, Some(stderr_path)).await.unwrap();
```

Comment:
On a follow-up release we will make this a bit more idiomatic to avoid booleans as raw function arguments :)